python/modules/IcePy/Operation.cpp

```diff
@@ -2787,12 +2781,9 @@ IcePy::TypedServantWrapper::ice_invokeAsync(
             }
         }
 
-        //
-        // See bug 4976.
```

Comment:
Corresponds to ICE-4527 in Jira.

The issue is/was that 3.4.1 versions of AMI ice_xxx operations incorrectly sent Normal instead of Nonmutating in some languages. The new checkNonIdempotent logic fixes this issue too.